lsp-ui-peek.el

```diff
@@ -698,12 +698,18 @@ references.  The function returns a list of `ls-xref-item'."
 Returns item(s)."
   (-when-let* ((locs (lsp-request method params))
                (locs (if (listp locs) locs (if (vectorp locs) (append locs nil) (list locs)))))
-    (mapcar #'lsp-ui-peek--get-xrefs-list
-            (if (gethash "uri" (car locs))
-                ;; Location[]
-                (--group-by (lsp--uri-to-path (gethash "uri" it)) locs)
-              ;; LocationLink[]
-              (--group-by (lsp--uri-to-path (gethash "targetUri" it)) locs)))))
+    (-filter
+     (-lambda ((&plist :file))
+       (or (f-file? file)
+           (prog1 nil
```

Comment:
`(prog1 nil` -> `(ignore`

Reply:
I wonder which one is the recommended solution here: `ignore` is shorter/more in a functional style and slower while prog1 is more imperative/faster and longer. I am interested to hear what elisp experts recommend.

Reply:
Why is prog1 faster ? No elisp expert at all :P

Reply:
prog1 is primitive written in C and there is no function invocation.